docs/reference/ccr/overview.asciidoc

```diff
@@ -1,7 +1,180 @@
 [role="xpack"]
 [testenv="platinum"]
-[[ccr-overview]]
+[[ccr-ov]]
 == Overview
 
```

Comment:
It would be better for SEO purposes to leave "overview" spelled out in this anchor. The subsections can use the abbreviation.